Doc/whatsnew/3.14.rst

```diff
@@ -421,6 +421,11 @@ typing
 * Remove :class:`!typing.ByteString`. It had previously raised a
   :exc:`DeprecationWarning` since Python 3.12.
 
+unicodedata
```

Comment:
This is the removal section. Please move to the improved modules section, around line 354.

Reply:
good catch